index.html

```diff
@@ -202,6 +202,23 @@ <h3>Open Demographics</h3>
 					<p>Open Demographics is a recommended set of questions that anyone can use to ask community members about their demographics.</p>
 				</a>
 			</div>
+			<div class="row">
+				<a class="col-sm-6 col-md-4 p-4" href="https://www.mediawiki.org/wiki/Wiki_Techstorm" target="_blank" rel="noopener">
+					<img alt="Wikimedia techstorm hackathon image link" src="img/projects/Techstorm-2019-logo-color.svg" />
+					<h3>Wikitechstorm</h3>
+					<p>Technical wikimedia hackaton designed initially for women*.</p>
+				</a>
+				<a class="col-sm-6 col-md-4 p-4" href="https://meta.wikimedia.org/wiki/Gender_gap" target="_blank" rel="noopener">
+					<img alt="Mind the gap image link" src="img/projects/Mind_the_gap1.svg" />
+					<h3>Gender gap</h3>
+					<p>Hub for resources and information about Wikimedia's gender gaps.</p>
+				</a>
+				<a class="col-sm-6 col-md-4 p-4" href="https://fr.wikipedia.org/wiki/Projet:Les_sans_pagEs" target="_blank" rel="noopener">
+					<img alt="les sans pages image link" src="img/projects/Les_sans_pagEs_drawing_Fhala.K.svg" />
```

Comment:
added alt text

Reply:
Hi this is all fine by me. Sorry for my late answer, I am not familiar with github and had lost my account when I changed my mobile number. What do I need to do now ?

Reply:
It's all good. Thank you for your work @Ahcatan. We'll merge now.

Reply:
sorry @Ahcatan I missed your reply thanks for following up here and thanks @Roshanjossey for merging :)